### PR TITLE
docker: no debug mode for docker files

### PR DESCRIPTION
**Pull request by crepererum**
* SECURITY Disables debug mode when using standard Docker image. Uses
  docker compose to set the variable instead.

* Adds note about running tests in debug config to documentation.

Signed-off-by: Marco Neumann <marco@crepererum.net>

### CODE REVIEW

File: docs/developers/docker.rst [hunk 1]
```diff
@@ -104,6 +104,11 @@ finished setup and the webservice is running. Then use:
 
     docker exec -it invenio_web_1 python setup.py test
 
+.. NOTE::
```
Comment (jirikuncar) [inline docs/developers/docker.rst]: `.. NOTE::` vs. `.. note::` ?

Comment (crepererum) [inline docs/developers/docker.rst]: Doesn't matter, sphinx gets both. The rest of the file uses the uppercase version.

**Comment by jirikuncar** [comment]
```console
$ git grep '.. NOTE::' | wc -l
1
$ git grep '.. note::' | wc -l
34
```

**Comment by crepererum** [comment]
*Making @jirikuncar happy*...**DONE** :wink:

**Pull request merged**